doc/topics/windows/windows-package-manager.rst

```diff
@@ -969,14 +945,18 @@ Firefox.
         salt-minion-py3:
             3001
 
+On running  ``pkg.list_pkgs``,  If any of the software installed on the machine matches the full name
```

Comment:
**Nitpick:** "If" should not be capitalized here.

Reply:
Updated